setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
```

Comment:
Could you describe the purpose then?  I don't see why this is not a good idea, moreover this variable is now checked when setting up the `vfatqc-python-scripts` repo:

https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/master/setup/paths.sh#L30

Based on the comments above it looks like the implementation is following the:

>  it should already be set up elsewhere, this script should simply tell the user that they have not done other things in life correctly

So this variable again seems necessary.